docs/src/pages/[platform]/connected-components/authenticator/customization/formfields/orderformfields/form-fields.angular.mdx

```diff
@@ -52,4 +53,8 @@ export class AppComponent {
     },
   }
 }
+
+// Adding the signUpAttributes prop is not needed since attributes are normally inferred via Zero Configuration.
+// For the sake of this example they have been explicitly added so you can copy and paste this into your own application and see it work. 
```

Comment:
Added additional not for this attribute, per offline conversation with @reesscot

Reply:
```suggestion
// Adding the signUpAttributes prop is typically not needed since attributes are inferred via Zero Configuration.
// For the sake of this example they have been explicitly added so you can copy and paste this into your own application and see it work. 
```

because not all customers might use CLI (e.g. https://github.com/aws-amplify/amplify-ui/issues/3196) imo. What do you think?

Reply:
Sure I can change to `typically` [876be16](https://github.com/aws-amplify/amplify-ui/pull/3172/commits/876be16062c506a1a6dd2f313be2274ab2d1d862)

Reply:
Oh, I meant putting that earlier in the sentence, like

```diff
- Adding the signUpAttributes prop is not needed since attributes are typically inferred via Zero Configuration.
+ Adding the signUpAttributes prop is typically not needed since attributes are inferred via Zero Configuration.
```

Reply:
OK, I think this is what you meant. [63cf4c6](https://github.com/aws-amplify/amplify-ui/pull/3172/commits/63cf4c61af047e741a3a2537267074b45d67a94b)